test/integration/freetext_editor_spec.js

```diff
@@ -1262,4 +1262,88 @@ describe("FreeText Editor", () => {
       );
     });
   });
+
+  describe("FreeText (remove)", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("tracemonkey.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must delete invisible annotations", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.click("#editorFreeText");
+          let currentId = 0;
+          const oneToFourteen = [...new Array(14).keys()].map(x => x + 1);
```

Comment:
While it won't matter here, since it's only a test, I do believe that this causes some unneeded intermediate iteration/allocation.
```suggestion
          const oneToFourteen = Array.from(new Array(14).keys(), x => x + 1);
```